### PR TITLE
Resolves #2157

### LINKED ISSUE #2157: typo in sentence
Typo / missing word here:
"A DNS server can load an existing scope its data file"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ea60398b-f9d6-f5c8-9228-6341e3cfd60b
* Version Independent ID: 7c8d32f9-f693-6b5c-0165-c1de361c04e8
* Content: [Add-DnsServerZoneScope (DnsServer)](https://docs.microsoft.com/en-us/powershell/module/dnsserver/add-dnsserverzonescope?view=win10-ps)
* Content Source: [docset/windows/dnsserver/Add-DnsServerZoneScope.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/dnsserver/Add-DnsServerZoneScope.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @AndreaBarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Typo.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2157

### CODE REVIEW

**Comment by e0i** [comment]
Closing and re-opening to refresh checks

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/windows/dnsserver/Add-DnsServerZoneScope.md [hunk 1]
```diff
@@ -33,16 +33,16 @@ Add-DnsServerZoneScope [-ZoneName] <String> [-Name] <String> [-LoadExisting] [-P
 
 ## DESCRIPTION
 The **Add-DnsServerZoneScope** cmdlet adds a zone scope to an existing zone on the Domain Name System (DNS) server.
-A DNS server can load an existing scope its data file.
+A DNS server can load an existing scope from its data file.
 The name of the scope should adhere to the same conventions as the zone name.
 The scope name cannot be same as the zone name to which this cmdlet adds it.
 
-When a scope is created, its data is persisted in a file named %Systemroot%\system32\dns\\\<zone name\>\\\<scope name\>.dns.
+When a scope is created, its data is persisted in a file named `%Systemroot%\system32\dns\\\<zone name\>\\\<scope name\>.dns`.
```
Comment (tseward) [inline docset/windows/dnsserver/Add-DnsServerZoneScope.md:40]: You can remove the escape characters since you're using ticks around the path.

**Review by tseward — CHANGES_REQUESTED** [review]

**Pull request merged**